index.bs

```diff
@@ -715,6 +716,13 @@ A [=sensor=] has an associated <dfn>periodic reporting mode flag</dfn> which is
 
 A [=sensor=] has an associated <dfn>current sampling frequency</dfn> which is initially null.
 
+The [=current sampling frequency=] for a non[=set/is empty|empty=] [=ordered set|set=] of
+[=activated sensor objects=] is an <dfn>optimal sampling frequency</dfn> which is estimated considering
+{{[[desiredSamplingFrequency]]|desired sampling frequencies}} of [=activated sensor objects|activated=]
+{{Sensor|Sensors}} and sampling frequency bounds defined by the underlying hardware.
```

Comment:
"underlying platform" is more appropriate term (here and in other places within the spec), as we actually use the limits set by the operating system not the actual hardware.